src/cpp/include/openvino/genai/perf_metrics.hpp

```diff
@@ -96,13 +100,15 @@ struct OPENVINO_GENAI_EXPORTS MeanStdPair {
  */
 struct OPENVINO_GENAI_EXPORTS PerfMetrics {
```

Comment:
btw, these metrics are text generation specific, while they don't work for VLM, Whisper and SD, because stages of the pipeline are different. Maybe we can have metrics as map of key and values? where some keys are common